### PR TITLE
glib: manually implement ParamSpec::is_valid_name

### PR DESCRIPTION
**Pull request by bilelmoussaoui**

### CODE REVIEW

**Comment by bilelmoussaoui** [comment]
Ok I'm lost here, the `g_param_spec_is_valid_name`  definitely exists on the ffi file and with the correct guard flag.

**Comment by sdroege** [comment]
> Ok I'm lost here, the `g_param_spec_is_valid_name` definitely exists on the ffi file and with the correct guard flag.

See my fixup. The `Cargo.toml` change was missing.

File: glib/src/param_spec.rs [hunk 1]
```diff
@@ -205,6 +211,36 @@ impl ParamSpec {
     //pub fn steal_qdata(&self, quark: /*Ignored*/glib::Quark) -> /*Unimplemented*/Option<Fundamental: Pointer> {
     //    unsafe { TODO: call gobject_ffi::g_param_spec_steal_qdata() }
     //}
+
+    #[doc(alias = "g_param_spec_internal")]
+    pub fn internal(
```
Comment (sdroege) [inline glib/src/param_spec.rs:216]: I think exposing it like this is not very useful. You'd have to provide some more infrastructure like for `GObject`s, and also expose `g_param_type_register_static()` / `GParamSpecTypeInfo` in some way (by providing some trait, I guess).

Comment (bilelmoussaoui) [reply glib/src/param_spec.rs:216]: Right, I don't plan on doing so as I don't really need it. I guess we can keep it and whoever needs that can provide that infra or drop the commit, whatever you prefer

Comment (sdroege) [reply glib/src/param_spec.rs:216]: I would drop it as whatever we're going to expose here will look different

**Comment by sdroege** [comment]
You dropped the `Cargo.toml` change

**Review by sdroege — APPROVED** [review]

**Pull request merged**